flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/log/CygnusLogger.java

```diff
@@ -67,7 +96,14 @@ public void fatal(String componentName, String message) {
      * @param message
      */
     public void warn(String componentName, String message) {
-        warn("[" + componentName + "] " + message);
+        try {
+            warn("[" + componentName + "] " + message);
+        } catch (Exception e) {
+            if (shutdown) {
+                System.err.println("A problem with the logging system was found... shutting down Cygnus right now!");
```

Comment:
Maybe you should also print the Exception trace/information to provide more information to the administrator reading the mesage?